Resources/Prototypes/wizardsDenWhitelists.yml

```diff
@@ -15,17 +15,17 @@
   - !type:ConditionNotesPlaytimeRange # Deny for >=2 medium severity notes in the last 14 days
```

Comment:
You need to update the comment as well to mention the 90 days instead of 14 days and >=3 medium severity notes instead of 2.